firebase/firebase-cpp-sdk
# Feature/tvos support integration tests storage

Changes done to XCode project for storage that allows us to run integration tests on device and simulator.

added a tvos target to XCode project for storage.
updated Podfile to build pods for tvos platform for this newly added target
To test this,

Make sure to have a valid GoogleService-Info.plist.
pod install
open integration_test.xcworkspace
Drag and drop firebase.xcframework, firebase_auth.xcframework and firebase_storage.xcframework

All tests pass on simulator but one (TestPutFileAndGetFile) is failing on device ( internal ticket: b/91209621).
